src/libsyntax/attr.rs

```diff
@@ -367,15 +494,7 @@ pub fn sort_meta_items(items: Vec<P<MetaItem>>) -> Vec<P<MetaItem>> {
     v.sort_by(|&(ref a, _), &(ref b, _)| a.cmp(b));
 
     // There doesn't seem to be a more optimal way to do this
-    v.into_iter().map(|(_, m)| m.map(|Spanned {node, span}| {
-        Spanned {
-            node: match node {
-                MetaItemKind::List(n, mis) => MetaItemKind::List(n, sort_meta_items(mis)),
-                _ => node
-            },
-            span: span
-        }
-    })).collect()
+    v.into_iter().map(|(_, m)| sort_meta_item(m)).collect()
```

Comment:
(now that they can contain literals, sorting `MetaItemKind::List`s might not even make sense)